features/build_runner/build_runner_output_row.rb

```diff
@@ -0,0 +1,44 @@
+module FastlaneCI
+  # Represents a single "Row" when streaming the build output (might be coming from fastlane)
+  # meaning it's a single message that we want to show to the user
+  # and store as a build artifact as part of the build log
+  class BuildRunnerOutputRow
+    BUILD_FAIL_TYPES = [:user_error, :build_error, :crash, :shell_error, :build_failure, :test_failure, :abort].to_set
+
+    # The type of message (e.g. `:message`, `:error`, `:important`)
+    attr_accessor :type
+
+    # The raw message to show
+    attr_accessor :message
+
+    # Timestamp (Time)
+    attr_accessor :time
+
+    # The plan is to remove this, as we want to render it on the front-end
+    attr_accessor :html
```

Comment:
Should we have a `TODO:` on here so that we check it before we release?

Reply:
We have a GitHub issue for this https://github.com/fastlane/ci/issues/213 I'll update the comment to link to it. No need for a TODO if we have a GitHub issue I think, especially since this is a not a blocking thing, like if we ship the first beta with this, it's fine, no security risk, and no issue with migrating to something new